### PR TITLE
beta 5.13.0 release-notes

### PR DESCRIPTION
**Pull request by ScottSuarez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,37 @@
...
+* provider: updated google.golang.org/api from v0.154.0 to v0.156.0 ([#6856](https://github.com/hashicorp/terraform-provider-google-beta/pull/6856))
+* provider: added support for in-place update for `labels` and `terraform_labels` fields in immutable resources ([#6857](https://github.com/hashicorp/terraform-provider-google-beta/pull/6857))
+
+FEATURES:
+* **New Resource:** `google_netapp_backup_policy` ([#6839](https://github.com/hashicorp/terraform-provider-google-beta/pull/6839))
+* **New Resource:** `google_netapp_volume` ([#6852](https://github.com/hashicorp/terraform-provider-google-beta/pull/6852))
+* **New Resource:** `google_network_security_address_group_iam_*` ([#6859](https://github.com/hashicorp/terraform-provider-google-beta/pull/6859))
+* **New Resource:** `google_network_security_security_profile` ([#6868](https://github.com/hashicorp/terraform-provider-google-beta/pull/6868))
+* **New Resource:** `google_vertex_ai_feature_group_feature` ([#6861](https://github.com/hashicorp/terraform-provider-google-beta/pull/6861))
+* **New Resource:** google_discovery_engine_data_store ([#6853](https://github.com/hashicorp/terraform-provider-google-beta/pull/6853))
```
Comment (ScottSuarez) [inline CHANGELOG.md:14]:
```suggestion
```
https://github.com/hashicorp/terraform-provider-google-beta/commits/release-5.13.0/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,36 @@
 ## 5.13.0 (Unreleased)
+NOTES:
+* cloudbuildv2: `google_cloudbuildv2_repository` now uses mmv1 engine instead of DCL ([#6843](https://github.com/hashicorp/terraform-provider-google-beta/pull/6843))
+* gkehub: updated beta api endpoint from v1beta1 to v1beta ([#6840](https://github.com/hashicorp/terraform-provider-google-beta/pull/6840))
```
Comment (ScottSuarez) [inline CHANGELOG.md:4]:
```suggestion
```

reverting https://github.com/GoogleCloudPlatform/magic-modules/pull/9846

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,36 @@
 ## 5.13.0 (Unreleased)
+NOTES:
+* cloudbuildv2: `google_cloudbuildv2_repository` now uses mmv1 engine instead of DCL ([#6843](https://github.com/hashicorp/terraform-provider-google-beta/pull/6843))
```
Comment (slevenick) [inline CHANGELOG.md:3]: Can this be in the form of the previous migration note:
compute: changed underlying actuation engine for google_network_firewall_policy and google_region_network_firewall_policy, there should be no user-facing impact

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,36 @@
...
+* bigquery: supported replica materialized view in resource_bigquery_table (beta) ([#6865](https://github.com/hashicorp/terraform-provider-google-beta/pull/6865))
+* cloudrunv2: added `nfs`, `gcs` fields to `google_cloud_run_v2_service.template.volumes` ([#6845](https://github.com/hashicorp/terraform-provider-google-beta/pull/6845))
+* cloudrunv2: added `tcpSocket` field to `google_cloud_run_v2.template.containers.liveness_probe`([#6845](https://github.com/hashicorp/terraform-provider-google-beta/pull/6845))
+* composer: added fields `enable_private_environment` and `enable_private_builds_only` to `google_composer_environment` ([#6870](https://github.com/hashicorp/terraform-provider-google-beta/pull/6870))
+* compute: added `enable_confidential_compute` field to `google_compute_instance.boot_disk.initialize_params` ([#6842](https://github.com/hashicorp/terraform-provider-google-beta/pull/6842))
+* compute: promoted `google_compute_disk.enable_confidential_compute` to GA ([#6842](https://github.com/hashicorp/terraform-provider-google-beta/pull/6842))
```
[the remark this answers is not in the document]
Comment (roaks3) [reply CHANGELOG.md:22]: Looks like that's what we have in GA, I will remove from the beta release notes

**Review by roaks3 — APPROVED** [review]

**Pull request merged**